src/components/Select/AutosuggestMulti/AutosuggestMulti.tsx

```diff
@@ -0,0 +1,150 @@
+                {suggestionToString(item)}
+            </SuggestionTag>
+        ));
+
+        if (numberOfHiddenTags > 0) {
+            const counter = (
+                <SuggestionTag key="counter" isBounded>{`+${numberOfHiddenTags}`}</SuggestionTag>
+            );
+            return [...shownTagsList, counter];
+        }
+
+        return shownTagsList;
+    };
+
+    const handleInputKeyDown = (blur: () => void) => (event: KeyboardEvent) => {
+        if (event.key === TAB_KEY) {
+            inputRef.current?.blur();
+            blur();
+        } else if (event.key === ESCAPE_KEY) {
+            // prevents key propagation and sets the focus on parent component
+            inputRef.current?.blur();
+            blur();
+            inputRef.current?.parentElement?.focus();
+            event.stopPropagation();
+        } else if (
+            event.key === BACKSPACE_KEY &&
+            !inputValue &&
+            selectedSuggestions &&
+            !!selectedSuggestions.length
+        ) {
+            const lastItem = selectedSuggestions[selectedSuggestions.length - 1];
+            onSelectionChange(lastItem);
+        }
+    };
+
+    // eslint-disable-next-line react/display-name
+    const renderFocused: FocusedRendererHelpers<S> = ({ getInputProps, onBlur: blur }) => (
+        <div {...elem('wrapper', { isFocused: true })}>
+            {renderFullTagsList()}
+            <input
+                {...getInputProps({
+                    ref: inputRef,
+                    placeholder: inputPlaceholder,
+                    onKeyDown: handleInputKeyDown(blur),
+                    'data-lpignore': true,
+                    ...elem('input'),
+                })}
+            />
+        </div>
+    );
+
+    // eslint-disable-next-line react/display-name
+    const renderBlurred: BlurredRendererHelpers<S> = ({ getInputProps }) => (
+        <div {...elem('wrapper')}>
+            {renderShortTagsList()}
+            {selectedSuggestions.length === 0 && (
+                <input
+                    {...getInputProps({
+                        ref: inputRef,
+                        placeholder: inputPlaceholder,
+                        'data-lpignore': true,
+                        ...elem('input'),
+                    })}
+                />
+            )}
+        </div>
+    );
+
+    return (
+        <SelectBase
+            {...rest}
+            suggestions={suggestions}
+            suggestionToString={suggestionToString}
+            inputRef={inputRef}
+            onBlur={onBlur}
+            onSelectionChange={onSelectionChange}
+            onInputValueChange={handleInputValueChange}
+            listRenderer={(listProps) => (
+                <SuggestionsList {...listProps} useOptimizeRender={useOptimizeListRender} />
+            )}
+            focusedRenderer={renderFocused}
+            blurredRenderer={renderBlurred}
+            keepExpandedAfterSelection
```

Comment:
When I add suggestions, the list collapses for me. Is that intended? Seems to be in contrast with this prop.

Reply:
The list collapsed because it doesn't have suggestions but it is still expanded. if we delete `keepExpandedAfterSelection` then after selection input loses the focus.

Reply:
I see that this clearing of input only happens in the example implementation. But that is different from current behaviour. Maybe better to show it without suggestions being cleared.